txp/Pos/Txp/Core/Core.hs

```diff
@@ -50,13 +77,70 @@ mkTxProof UnsafeTxPayload {..} =
     { txpNumber = fromIntegral (length _txpTxs)
     , txpRoot = mtRoot _txpTxs
     , txpWitnessesHash = hash _txpWitnesses
-    , txpDistributionsHash = hash _txpDistributions
     }
 
 -- | Convert 'TxPayload' into a flat list of `TxAux`s.
 flattenTxPayload :: TxPayload -> [TxAux]
 flattenTxPayload UnsafeTxPayload {..} =
-    zipWith3 TxAux (toList _txpTxs) _txpWitnesses _txpDistributions
+    zipWith TxAux (toList _txpTxs) _txpWitnesses
 
 emptyTxPayload :: TxPayload
-emptyTxPayload = leftToPanic @Text "emptyTxPayload failed" $ mkTxPayload []
+emptyTxPayload = mkTxPayload []
+
+----------------------------------------------------------------------------
+-- Details
+----------------------------------------------------------------------------
+
+-- Compute bootstrap era distribution.
+--
+-- This function splits coin into chunks of @weightsSum@ and
+-- distributes them between bootstrap era stakeholders. Remainder is
+-- assigned randomly (among bootstrap era stakeholders) based on hash of the
```

Comment:
I guess that distributing the remainder among the stakeholders (instead of giving it all to one person) would be nicer, but I don't know whether it's important enough to bother implementing it.